.mvn/gradle-enterprise.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<gradleEnterprise
+        xmlns="https://www.gradle.com/gradle-enterprise-maven" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="https://www.gradle.com/gradle-enterprise-maven https://www.gradle.com/schema/gradle-enterprise-maven.xsd">
+    <server>
+        <url>https://ge.apache.org</url>
+        <allowUntrusted>false</allowUntrusted>
+    </server>
+    <buildScan>
+        <capture>
+            <goalInputFiles>true</goalInputFiles>
+            <buildLogging>true</buildLogging>
+            <testLogging>true</testLogging>
+        </capture>
+        <backgroundBuildScanUpload>#{isFalse(env['GITHUB_ACTIONS'])}</backgroundBuildScanUpload>
+        <publish>ALWAYS</publish>
```

Comment:
I would recommend adding `publishIfAuthenticated>true</publishIfAuthenticated>`. It is a hidden option, but will give unauthenticated users a better user experience, given that not every builder of this open source project will be an authenticated user to https://ge.apache.org.

Reply:
Thank you! It actually exists also in the template I copied from. But it seems this option is omitted from the schema..

Let me add it now.

Reply:
That is correct, it is not currently an option in the schema. It is an option that is only recommended for use in open source environments where it is not desirable for every single developer that clones the project to be able to authenticate and publish build scans.

You can see the [SCIMple project](https://github.com/apache/directory-scimple/blob/develop/.mvn/gradle-enterprise.xml#L37) for another example of usage of this option (as well as configuration of Gradle Enterprise generally).